test_aider.el

```diff
@@ -1,23 +1,118 @@
-
 (require 'ert)
 
-(ert-deftest aider-buffer-name-from-git-repo-path-test ()
```

Comment:
remove out of date tests and generate some new one for non interactive functions